tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/SystemDataSqlClientTests.cs

```diff
@@ -89,6 +89,7 @@ public async Task SubmitsTraces(string packageVersion, string metadataSchemaVers
             settings.AddSimpleScrubber("peer.service: localhost", "peer.service: sqlserver");
             settings.AddSimpleScrubber("peer.service: (localdb)\\MSSQLLocalDB", "peer.service: sqlserver");
             settings.AddSimpleScrubber("peer.service: sqledge_arm64", "peer.service: sqlserver");
+            settings.AddRegexScrubber(new Regex("dd.instrumentation.time_ms: \\d+.\\d+"), "dd.instrumentation.time_ms: 123.456");
```

Comment:
Shall we perhaps measure the instrumentation time in microseconds instead of milliseconds for higher precision, as database can have sub-milliseconds statements?

Reply:
it's saved as a double anyway, so we will have microsecond precision, I just though having the decimal point after the milliseconds was the most readable.